protocols/peer/README.md

```diff
@@ -113,12 +113,12 @@ function unsetRule(
 | `_takerToken` | `address` | The token that the Peer would send in a trade.     |
 | `_makerToken` | `address` | The token that the Consumer would send in a trade. |
 
-## Get a Buy Quote
+## Get a Maker-Side Quote
 
-Get a quote to buy from the Peer.
+Get a quote for the maker side. Often used to get a buy price for \_makerToken.
 
 ```Solidity
-function getBuyQuote(
+function getMakerSideQuote(
```

Comment:
For what reason did you think it was unclear before? I felt like buy and sell were clear

Reply:
Fair enough